Lib/concurrent/futures/process.py

```diff
@@ -732,5 +723,3 @@ def shutdown(self, wait=True, *, cancel_futures=False):
             self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
-
-atexit.register(_python_exit)
```

Comment:
This change is just to make the atexit registration be in the same location for both ProcessPoolExecutor and ThreadPoolExecutor (right under the definition for `_python_exit()`). It can be removed if needed, but I think it will make them easier to locate for those reading over the implementation details.